src/ASM/LR/ASMu3Dmx.h

```diff
@@ -36,7 +36,7 @@ class ASMu3Dmx : public ASMu3D, private ASMmxBase
   //! \brief Copy constructor.
   ASMu3Dmx(const ASMu3Dmx& patch, const CharVec& n_f = CharVec(3,0));
   //! \brief Empty destructor.
-  virtual ~ASMu3Dmx() {}
+  virtual ~ASMu3Dmx() = default;
```

Comment:
Guess these are equivalent, right? Just to be consistent.

Reply:
yes, they are equivalent.